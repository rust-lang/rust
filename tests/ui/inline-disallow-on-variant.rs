enum Foo {
    #[inline]
    //~^ ERROR attribute should be applied
    Variant,
}

fn main() {}
