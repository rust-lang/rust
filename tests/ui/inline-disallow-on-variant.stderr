error[E0518]: attribute should be applied to function or closure
  --> $DIR/inline-disallow-on-variant.rs:2:5
   |
LL |     #[inline]
   |     ^^^^^^^^^
LL |
LL |     Variant,
   |     ------- not a function or closure

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0518`.
